prow/cmd/crier/README.md

```diff
@@ -0,0 +1,60 @@
+# Crier
+
+Crier reports your prowjobs on their status changes.
+
+## Usage / How to enable existing available reporters
+
+### [Gerrit reporter](/prow/gerrit/reporter)
```

Comment:
Please specify the behavior of each reporter, i.e. what states are reported and in what way.

Reply:
good point, added.